### PR TITLE
DM-25689: fixes for Gen2->Gen3 repo conversion

### PR DESCRIPTION
**Pull request by TallJimbo**

### CODE REVIEW

File: python/lsst/obs/subaru/strayLight/formatter.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 class SubaruStrayLightDataFormatter(FileFormatter):
     """Gen3 Butler Formatters for HSC y-band stray light correction data.
     """
+    extension = ".fits"
```
Comment (laurenam) [inline python/lsst/obs/subaru/strayLight/formatter.py:32]: Can’t help but be curious...is the stutter (with L34) deliberate?

Comment (timj) [reply python/lsst/obs/subaru/strayLight/formatter.py:32]: No. I fixed it last week and the rebase didn’t notice so added the second one.

Comment (laurenam) [reply python/lsst/obs/subaru/strayLight/formatter.py:32]: Doh...!

Comment (TallJimbo) [reply python/lsst/obs/subaru/strayLight/formatter.py:32]: Gah, I saw this before merge, amended the commit to drop that part, and then somehow it came back before I merged.

**Pull request merged**